public/app/plugins/panel/canvas/ConnectionSVG.tsx

```diff
@@ -0,0 +1,221 @@
+        for (let c of v.options.connections) {
+          const target = c.targetName ? scene.byName.get(c.targetName) : v.parent;
+          if (target) {
+            connections.push({
+              source: v,
+              target,
+              info: c,
+            });
+          }
+        }
+      }
+    }
+    return connections;
+  };
+
+  // Figure out target and then target's relative coordinates drawing (if no target do parent)
+  const renderConnections = () => {
+    return findConnections().map((v, idx) => {
+      const { source, target, info } = v;
+      const sourceRect = source.div?.getBoundingClientRect();
+      const parent = source.div?.parentElement;
+      const parentRect = parent?.getBoundingClientRect();
+
+      if (!sourceRect || !parent || !parentRect) {
+        return;
+      }
+
+      const parentBorderWidth = parseFloat(getComputedStyle(parent).borderWidth);
+
+      const sourceHorizontalCenter = sourceRect.left - parentRect.left - parentBorderWidth + sourceRect.width / 2;
+      const sourceVerticalCenter = sourceRect.top - parentRect.top - parentBorderWidth + sourceRect.height / 2;
+
+      // Convert from connection coords to DOM coords
+      // TODO: Break this out into util function and add tests
+      const x1 = sourceHorizontalCenter + (info.source.x * sourceRect.width) / 2;
+      const y1 = sourceVerticalCenter - (info.source.y * sourceRect.height) / 2;
+
+      let x2;
+      let y2;
+
+      if (info.targetName) {
+        const targetRect = target.div?.getBoundingClientRect();
+
+        const targetHorizontalCenter = targetRect!.left - parentRect.left - parentBorderWidth + targetRect!.width / 2;
+        const targetVerticalCenter = targetRect!.top - parentRect.top - parentBorderWidth + targetRect!.height / 2;
+
+        x2 = targetHorizontalCenter + (info.target.x * targetRect!.width) / 2;
+        y2 = targetVerticalCenter - (info.target.y * targetRect!.height) / 2;
+      } else {
+        const parentHorizontalCenter = parentRect.width / 2;
+        const parentVerticalCenter = parentRect.height / 2;
+
+        x2 = parentHorizontalCenter + (info.target.x * parentRect.width) / 2;
+        y2 = parentVerticalCenter - (info.target.y * parentRect.height) / 2;
+      }
+
+      const isSelected = selectedConnection === info;
+      const selectedStyles = { stroke: '#44aaff', strokeWidth: 3 };
+
+      return (
+        <svg className={styles.connection} key={idx}>
+          <g onClick={() => selectConnection(info, source)}>
+            <defs>
+              <marker
+                id={CONNECTION_HEAD_ID}
+                markerWidth="10"
+                markerHeight="7"
+                refX="10"
+                refY="3.5"
+                orient="auto"
+                stroke="rgb(255,255,255)"
+              >
+                <polygon points="0 0, 10 3.5, 0 7" fill="rgb(255,255,255)" />
+              </marker>
+            </defs>
+            <line
```

Comment:
```suggestion
            <line
              id={`${CONNECTION_LINE_ID}_transparent`}
              cursor="grab"
              stroke="transparent"
              pointerEvents="auto"
              strokeWidth={30}
              x1={x1}
              y1={y1}
              x2={x2}
              y2={y2}
            />
            <line
```

If we add another transparent line that is wider, selecting becomes easier.

Reply:
great suggestion - I was thinking in a similar way to improve UX of selecting arrow as right now it is a little painful